src/plugins/MessagesPlugin/Messages.ts

```diff
@@ -0,0 +1,43 @@
+import MessagesRepository from '@/plugins/MessagesPlugin/MessagesRepository';
+import MessageKeys from '@/plugins/MessagesPlugin/MessageKeys';
+import { InjectionKey } from 'vue';
+
+/**
+ * Usage (assuming this has been registered as a Vue plugin):
+ *
+ * FIXME: write something about when to use getText vs get
+ *
+ * `this.$messages.get( this.$messages.KEYS.LEMMA_INPUT_LABEL )`
+ * `this.$messages.getText( this.$messages.KEYS.LEMMA_INPUT_LABEL )`
+ */
+export default class Messages {
+
+	public readonly KEYS = MessageKeys;
+
+	private readonly messagesRepository: MessagesRepository;
+
+	public constructor( messagesRepository?: MessagesRepository ) {
+		if ( !messagesRepository ) {
+			this.messagesRepository = {
+				get( messageKey ) {
+					return `⧼${messageKey}⧽`;
+				},
+				getText( messageKey ) {
+					return `⧼${messageKey}⧽`;
+				},
+			};
+			return;
+		}
+		this.messagesRepository = messagesRepository;
+	}
+
+	public get( messageKey: string, ...params: readonly ( string|HTMLElement )[] ): string {
+		return this.messagesRepository.get( messageKey, ...params );
+	}
+
+	public getUnescaped( messageKey: string, ...params: readonly string[] ): string {
```

Comment:
Unescaped in this context means the raw text string? What is the use case example for unescaped text if we don't plan on using it as `innerHTML`?

Reply:
We want to use this in templates directly with `{{}}`, e.g. `<label>{{ messages.getUnescaped( 'label-text ) }}</label>` where it would be double-escaped with just using `.get()`.

Reply:
Aha, I see, thank you!